### ISSUE TITLE
bookmarks-plus mirror is almost a month out of date

### ISSUE THREAD

**Issue by nealmcb** [emacsmirror/p.old#1]
Thank you so much for hosting this mirror. It's much easier to manage things this way!

But at least one package mirror seems out of date: 

 https://github.com/emacsmirror/bookmark-plus

The latest there is from July 17, but there were perhaps 6 more revisions between 2012 July 21 and Aug 10 according to

http://www.emacswiki.org/emacs/?action=history;id=icicles-chg.el

http://www.emacswiki.org/emacs/icicles-chg.el

**Comment by tarsius** [comment]
bookmark+ and icicles should be up-to-date again, the other packages are being updated right now.

Normally I update the packages from the Emacswiki about once a week, sometimes a little less often. The current delay is in part due to a recent problem on the Emacswiki. Once that was resolved I delayed updating to see if was fixed for good. And then there was a bug in my import code... Thanks for pocking me :-)
